Cargo.toml

```diff
@@ -1,16 +1,40 @@
 [package]
 name = "hades252"
-version = "0.1.0"
-authors = ["kev <kevtheappdev@gmail.com>"]
+version = "0.1.1"
```

Comment:
Since the API changes completely, this should change a major version to indicate that the compatibility is broken. But since was not public maybe it's not needed.

Reply:
I thought about that, if we had follow semver now, it would have been a breaking change and therefore a `major` version as you said. But having `1.0.0` would have meant that it's actually stable and ready to release – and we're still in the unstable phase.
I therefore applied the minimum version bump, but we might decide also to go from `0.1.0` to `0.2.0`, I would just avoid bump the major.

Reply:
Perfect and totally undestandable!